tests/integration_tests/test_ingest_component.py

```diff
@@ -90,6 +100,54 @@ def test_ingest_with_r4_file_success(test_client_function, towns_fund_round_4_fi
     }
 
 
+def test_ingest_with_r4_file_success_with_load(test_client_function, towns_fund_round_4_file_success):
+    """Tests that, given valid inputs, the endpoint responds successfully."""
+    endpoint = "/ingest"
+    response = test_client_function.post(
+        endpoint,
+        data={
+            "excel_file": towns_fund_round_4_file_success,
+            "reporting_round": 4,
+            "place_names": ["Blackfriars - Northern City Centre"],
+            "do_load": True,
+        },
+    )
+
+    assert response.status_code == 200, f"{response.json}"
+    assert response.json == {
+        "detail": "Spreadsheet successfully validated and ingested",
+        "loaded": True,
+        "metadata": {
+            "FundType_ID": "HS",
+            "Organisation": "Worcester City Council",
+            "Programme ID": "HS-WRC",
+            "Programme Name": "Blackfriars - Northern City Centre",
+        },
+        "status": 200,
+        "title": "success",
+    }
```

Comment:
I'm not sure there's much value in having a function for both ```do_load=True``` and ```do_load=False``` on a successful file. The only reason I turned off the load parameter for that test was to save time.  ```do_load=True``` is probably a more illustrative use case so we can remove the previous ```test_ingest_with_r4_file_success``` function I think.